packages/sdk/src/index.ts

```diff
@@ -1,6 +1,11 @@
-import { LinearWebhookClient } from "./webhooks";
+import { LinearWebhookClient } from "./webhooks/index.js";
 
-export * from "./index_umd";
```

Comment:
:wat:  didn't realize we had this before, nice to see this removed haha

Reply:
Yea I guess it made sense at the time, but definitely not really needed anymore